pom.xml

```diff
@@ -683,7 +683,7 @@
         <dependency>
           <groupId>io.cdap.tests.e2e</groupId>
           <artifactId>cdap-e2e-framework</artifactId>
-          <version>0.1.0-SNAPSHOT</version>
+          <version>0.2.0-SNAPSHOT</version>
```

Comment:
revert it back to 0.1.0-SNAPSHOT

Reply:
This Snapshot version has changed by @Vipinofficial11

Reply:
change it back

Reply:
@vikasrathee-cs This change is with the e2e tests fixes, I have updated the snapshot version to latest.